README.md

```diff
@@ -46,20 +46,42 @@ POSIX threads library needs LSE atomic instructions.  LSE is important for
 locking and thread synchronization routines.  The dynamic linker on Linux can
 detect CPU capabilities and load libc built with LSE.  For example, Ubuntu 20.04
 contains the libc6-lse package.  When installed on a Graviton2 system, all
-applications will link against this library.  ldd command line utility displays
-the path of the linked dynamic libraries.  If libpthreads.so is in atomics
-directory, the library uses LSE atomic instructions.  Amazon Linux 2 will soon
-provide a similar mechanism to select an LSE version of libc.
+applications will link against this library.  __Action item: check that an
+application links against an LSE enabled library.__  `ldd` command line utility
+displays the path of the linked dynamic libraries.  If libpthreads.so is in
+atomics directory, the library uses LSE atomic instructions.  Amazon Linux 2
+will soon provide a similar mechanism to select an LSE version of libc.
 
 The compiler needs to generate LSE instructions for applications that use atomic
 operations.  For example, the code of databases like PostgreSQL contain atomic
 constructs; c++11 code with std::atomic statements translate into atomic
-operations.  GCC's -march=armv8.2-a flag enables all instructions supported by
-Graviton2, including LSE.  Binaries compiled with this flag won’t work on
-Graviton powered A1 instances.  GCC's -moutline-atomics flag produces a binary
-that runs on both Graviton and Graviton2.  Supporting both platforms with the
-same binary comes at a small extra cost: one load and one branch.  CPU branch
-predictor lowers the overhead when the extra branch is in hot code.
+operations.  GCC's `-march=armv8.2-a` flag enables all instructions supported by
+Graviton2, including LSE.  __Action item: check that an application uses LSE
```

Comment:
Instead of "Action item" How about, "To confirm that LSE instructions are being created..."

Reply:
done